### PR TITLE
Docupdates

### PR DESCRIPTION
**Pull request by tonyleachsf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,13 @@ The PayPal Here SDK enables iOS apps to interact with credit card swipers so tha
 * **Interact with PayPal Hardware** — Detect, connect to, and listen for events coming from PayPal Here audio jack-based card swipers.
 * **Process Card-Present payments** — When you swipe a card through a PayPal Here swiper, card data is immediately encrypted. The encrypted package can be sent to PayPal alongside the transaction data for processing.
 
+Developers should use the PayPal Here SDK to get world-class payment process with extremely simple integration.  Some of the main benefits include
+* **Low, transparent pricing:** Merchants pay just 2.7% per transaction (or 3.5% + $0.15 for keyed in transactions), including cards like American Express, with no additional hidden/monthly costs.
```
Comment (djMax) [inline README.md]: This is US only pricing... should be noted.

File: README.md [hunk 2]
```diff
@@ -202,29 +186,44 @@ if(response.isSignatureRequiredToFinalize) {
...
+ [self showPaymentCompeleteView:_capturedPaymentResponse];
+ }];
 ```
+(SignatureViewController.m)
 
-Required signatures and the PPHTransactionManager
-=================================================
+More Stuff to Look At
+=====================
+We've just scratched the surface of what's available in the PayPal Here SDK.  More detail is available in our [developer documentation](/docs/DeveloperGuide_iOS.pdf) to show other capabilities.  These include:
```
Comment (djMax) [inline README.md]: I'd leave out the "we've just scratched the surface" bit and just say "much more is available" or something less prideful. :)

Comment (tonyleachsf) [inline README.md]: done.

File: README.md (hunk 1 above)
Comment (tonyleachsf) [inline README.md]: done.

**Comment by pphkash** [comment]
Document updates look great.

**Pull request merged**